eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/AlonzoEraGen.hs

```diff
@@ -415,11 +415,12 @@ instance Mock c => EraGen (AlonzoEra c) where
           Set.empty
           mapScriptWit
           -- (dataMapFromTxOut (Prelude.foldr (:) [] (outputs' txbody)) (TxDats (getDataMap scriptinfo mapScriptWit)))
-          (dataMapFromTxOut (Prelude.foldr (:) [] (Map.elems smallUtxo)) (TxDats (getDataMap scriptinfo mapScriptWit)))
+          (dataMapFromTxOut smallUtxo (TxDats (getDataMap scriptinfo mapScriptWit)))
           -- The data hashes come from two places
           (Redeemers rdmrMap)
       txinputs = inputs' txbody
-      smallUtxo = eval (txinputs ◁ utxo) :: Map.Map (TxIn c) (Core.TxOut (AlonzoEra c))
+      smallUtxo :: [Core.TxOut (AlonzoEra c)]
+      smallUtxo = SplitMap.elems (unUTxO utxo `SplitMap.restrictKeysSet` txinputs)
```

Comment:
Perhaps like this

smallUtxo = (txinputs Split.◁ utxo)

Reply:
This is a test suite, so we don't need to match any specs here. I'll leave it as is.